src/TRegx/DataProvider/CrossDataProviders.php

```diff
@@ -3,6 +3,11 @@
 
 class CrossDataProviders
 {
+    /**
+     * @param array<array-key, array<int, mixed>> ...$dataProviders
+     *
+     * @return array<string, array<int, mixed>>
```

Comment:
Thinking of this: I think it needs to be `int|string`, not just `string`. Change coming up in a minute.

Reply:
Ah, no, for the `@return`, the keys always are strings. But I've made the `@param` annotations a bit more specific and repushed.